api/v1/qdrantcluster_types.go

```diff
@@ -693,17 +703,22 @@ type QdrantClusterStatus struct {
 	// +optional
 	AvailableNodeIndexes []int `json:"availableNodeIndexes,omitempty"`
 	// BootstrapNode specifies the node in the cluster which will be used for bootstrapping a new node.
-	// Should be 0...n-1 (default = 0)
+	// Should be a value from AvailableNodeIndexes.
+	// As default the value from AvailableNodeIndexes[0] will be used.
 	// +optional
 	BootstrapNode int `json:"bootstrapNode,omitempty"`
 	// If set the operator will scale down 1 peer and reset the bool.
 	// If you want to remove more then 1 peer, you need to repeat setting this bool
 	// +optional
 	ScaleDownAllowed bool `json:"scaleDownAllowed,omitempty"`
+	// The node index used in a scale down (see ScaleDownAllowed)
+	// If this field is not set the last index in AvailableNodeIndexes will be used.
```

Comment:
who is setting this field? The operator?

Reply:
Yes (once the cluster-manager instructs the operator to do). Or alternative (not recommended, if the CM isn't enabled via a [runbook](https://www.notion.so/qdrant/Down-Scale-cluster-horizontal-180989427f91435ba1b9763358d61b8f))